server.js

```diff
@@ -66,55 +64,43 @@ app.use('/', (req, res, next) => {
   return next();
 });
 
+app.use('/*', (req, res, next) => {
+  const initialStore = Object.assign(initialState, { lastLoaded: req._parsedUrl.path });
```

Comment:
When you do this with `initialState` as the first argument, you are overriding the `lastLoaded` attribute. I'm not sure if this is actually what you want. You can do `Object.assign({}, initialState, { lastLoaded: req._parsedUrl.path });` or also use the spread operator which is more commonly used (but equivalent to Object.assign) `{ ...initialState, lastLoaded: req._parsedUrl.path };`.

Reply:
Thanks. I mixed up `Object.assign` and spread operator syntax.